src/integrationTest/java/uk/gov/hmcts/reform/cpo/utils/UIDService.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.cpo.utils;
+
+import org.springframework.stereotype.Service;
+
+import java.security.SecureRandom;
+
+@SuppressWarnings("PMD")
+@Service
+public class UIDService {
+
+    private final SecureRandom random = new SecureRandom();
+
+    /**
+     * Generates a random valid UID.
+     *
+     * @return A randomly generated, valid, UID.
+     */
+    public String generateUID() {
+        String currentTime10OfSeconds = String.valueOf(System.currentTimeMillis()).substring(0, 11);
+        StringBuilder builder = new StringBuilder(currentTime10OfSeconds);
+        for (int i = 0; i < 4; i++) {
```

Comment:
Could we use   Streams ?   for instance IntStream.of(1, 2, 3); or IntStream.iterate(0, i -> i).limit(3);
Just a suggestion.

Reply:
I can change yes, this is code copied directly from `ccd-datastore` so have tried not to modify it

https://github.com/hmcts/ccd-data-store-api/blob/3eab3125d66ddb87d1d3e4a2dc2032cb75b4be9c/src/main/java/uk/gov/hmcts/ccd/domain/service/common/UIDService.java#L20